Tests/scripts/convert_test_result_to_jira_issues.py

```diff
@@ -0,0 +1,183 @@
+import argparse
+import json
+import os
+import sys
+import traceback
+from datetime import datetime, timezone, timedelta
+from tempfile import NamedTemporaryFile
+
+import urllib3
+from jira.client import ResultList, Issue, JIRA
+from junitparser import TestSuite, JUnitXml
+from tabulate import tabulate
+
+from Tests.scripts.utils import logging_wrapper as logging
+from Tests.scripts.utils.log_util import install_logging
+
+urllib3.disable_warnings()  # Disable insecure warnings
+
+GITLAB_PROJECT_ID = os.getenv('CI_PROJECT_ID') or 2596  # the default is the id of the content repo in code.pan.run
+GITLAB_SERVER_URL = os.getenv('CI_SERVER_URL', 'https://code.pan.run')  # disable-secrets-detection
```

Comment:
Do we need to expose these internal values?

Reply:
it's exposed in all other scripts, i will remove it.